src/lucky_migrator/runner.cr

```diff
@@ -33,6 +35,14 @@ class LuckyMigrator::Runner
     if (message = e.message) && message.includes?(%("#{self.db_name}" already exists))
       puts "Already created #{self.db_name.colorize(:green)}"
       exit(0)
+    elsif (message = e.message) && (message.includes?("createdb: not found") || message.includes?("No command 'createdb' found"))
+      raise <<-ERROR
+      PostgreSQL is not installed.
```

Comment:
This may not be correct. For example, on macOS, you can have PostgresSQL but not the postgres tools. I think instead it would be good to reprint the error message here: `#{message}` so the user can see the message that was raised and google it if the suggestions below do not work

Reply:
No problem, honestly I wasn't absolutely sure about this message either so this works better.